src/super_gradients/training/sg_model/sg_model.py

```diff
@@ -1529,7 +1535,7 @@ def _initialize_sg_logger_objects(self):
 
             self.sg_logger = SG_LOGGERS[sg_logger](**sg_logger_params)
         else:
-            raise RuntimeError('sg_logger can be either an sg_logger name (str) or a subcalss of AbstractSGLogger')
+            raise RuntimeError('sg_logger can be either an sg_logger name (str) or an instance of AbstractSGLogger')
```

Comment:
Mistake I guess ?

Reply:
Was a mistake, just sneaking in a fix into this PR